### ISSUE TITLE
Face recognition (identification) after detection

### ISSUE THREAD

**Issue by schelian** [wg-perception/people#46]
Hi all, has anyone done this?  I am using the OpenFace and [its ROS wrapper](https://github.com/tue-robotics/openface_ros).  This wants a 2D image but I just want to pass it the ROI / bounding box where the face was first detected (plus a little area around it).

The messages output from this package are the 3D coordinate but the visualization part of it has an ROI / bounding box.  Should I change the message from this package to also output the ROI or is there an easier way?

**Comment by DLu** [comment]
I don't believe anyone is actively working on this, at least as part of this stack.
